packages/flutter/lib/src/services/text_input.dart

```diff
@@ -744,12 +744,17 @@ class RawFloatingCursorPoint {
   /// [FloatingCursorDragState.Update].
   RawFloatingCursorPoint({
     this.offset,
+    this.startLocation,
     required this.state,
   }) : assert(state != FloatingCursorDragState.Update || offset != null);
 
   /// The raw position of the floating cursor as determined by the iOS sdk.
   final Offset? offset;
 
+  /// (local offset, new caret location)
+  ///  Only non-null when starting a floating cursor via long press.
+  final (Offset, TextPosition)? startLocation;
```

Comment:
Can this just be an `Offset` and we can get the current value from within `updateFloatingCursor`?

Reply:
I had a discussion with LongCatIsLooong, and because `renderEditable.selection` updates one frame late, we can't use `renderEditable.selection` for calculation.

Reply:
Since this is public API documentation, could you update it to make it more formal / descriptive?